Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -365,34 +393,45 @@ extension ActivityStreamPanel {
                 withUserData: userData,
                 toApplication: UIApplication.sharedApplication())
         })
-
+        
         let deleteFromHistoryAction = ActionOverlayTableViewAction(title: Strings.DeleteFromHistoryContextMenuTitle, iconString: "action_delete", handler: { action in
-            ASOnyxPing.reportDeleteItemEvent(actionPosition: indexPath.item, source: .highlights)
+            ASOnyxPing.reportDeleteItemEvent(actionPosition: indexNumber, source: eventSource)
             self.profile.history.removeHistoryForURL(site.url)
         })
-
+        
         let shareAction = ActionOverlayTableViewAction(title: Strings.ShareContextMenuTitle, iconString: "action_share", handler: { action in
             if let url = NSURL(string: site.url) {
                 let helper = ShareExtensionHelper(url: url, tab: nil, activities: [])
                 let controller = helper.createActivityViewController { completed, activityType in
-                    ASOnyxPing.reportShareEvent(actionPosition: indexPath.row, source: .highlights, shareProvider: activityType)
+                    ASOnyxPing.reportShareEvent(actionPosition: indexNumber, source: eventSource, shareProvider: activityType)
```

Comment:
Farhan, I see these ASOnyxPing references in the latest version of the code. Looks like sleroux [re-committed](https://github.com/mozilla-mobile/firefox-ios/commits/master/Client/Frontend/Home/ActivityStreamPanel.swift) these.